packages/devtools_app/lib/src/screens/memory/panes/chart/widgets/chart_pane.dart

```diff
@@ -209,8 +201,8 @@ class _MemoryChartPaneState extends State<MemoryChartPane>
               // The legend.
               MultiValueListenableBuilder(
                 listenables: [
-                  widget.chartController.legendVisibleNotifier,
-                  controller.controllers.chart.isAndroidChartVisibleNotifier,
+                  widget.chart.legendVisibleNotifier,
+                  widget.chart.isAndroidChartVisible,
```

Comment:
Do we have a precedent on how we name notifiers? We should try and be consistent with out naming scheme.

Reply:
We use all possible ways. Made it consistent here.